subsys/net/lib/websocket/websocket_internal.h

```diff
@@ -100,11 +119,28 @@ __net_socket struct websocket_context {
 
 	/** Did we receive all from peer during HTTP handshake */
 	uint8_t all_received : 1;
-
-	/** Header received */
-	uint8_t header_received : 1;
 };
 
+#if defined(CONFIG_NET_TEST)
+	/**
+	 * Websocket unit test does not use socket layer but feeds
+	 * the data directly here when testing this function.
+	 */
+	struct test_data {
+		/** pointer to data "tx" buffer */
+		uint8_t *input_buf;
+
+		/** "tx" buffer data length */
+		size_t input_len;
+
+		/** "tx" buffer read (recv) position */
+		size_t input_pos;
+
+		/** external test context */
+		struct websocket_context *ctx;
+	};
```

Comment:
This structure doesn't need to be indented.

Reply:
Fixes